llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -53,7 +53,7 @@ constexpr char SPEC_CONST_DEFAULT_VAL_MD_STRING[] =
 
 void AssertRelease(bool Cond, const char *Msg) {
```

Comment:
If this function defined in several files then it might be defined as utils function in one place and just reused.

Reply:
Thank you @mlychkov The `AssertRelease` function is converted to a macro and extracted in a new
header file `Support.h`. The macro is required to let `__FILE__` be preprocessed in a real source file name, not in just `Support.h`.